types/express-mysql-session/index.d.ts

```diff
@@ -50,19 +58,17 @@ declare class MySQLStoreClass {
 
     length(callback?: (error: any, count: any) => void): void;
 
+    all(callback?: (error: any, count: any) => void): void;
+
     clear(callback?: (error: any) => void): void;
 
     clearExpiredSessions(callback?: (error: any) => void): void;
 
+    query(sql: string, params: any, callback?: (error: any, rows: any, fields: any) => void): void;
```

Comment:
Can we get better Typings for it?

Maybe:

```ts
query<T = any>(sql: string, params: any, callback?: (error: any, rows: Array<T>, fields: any) => void): void;
```

Something similar for fields...

Reply:
I've looked at the express-mysql-session and mysql code but I cannot figure out what the exact types of the rows and fields are in this situation, instead of making a mistake and breaking things, I'd rather set them to `any`